.github/actions/run-script-with-cache/action.yml

```diff
@@ -37,17 +41,24 @@ runs:
         path: '${{ github.workspace }}/${{ inputs.repository }}${{ inputs.cache-path }}'
         key: '${{ inputs.cache-key }}-${{ steps.get-date.outputs.today }}'
     - name: Run script
-      shell: bash
+      shell: bash -l {0}
       working-directory: ${{ inputs.repository }}
       env:
         SCRIPT: ${{ inputs.script }}
       if: ${{ inputs.cache-path == '' || steps.run-cache-action.outputs.cache-hit != 'true' }}
       run: |
-        set -euxo pipefail
+        if [[ ${{ inputs.is_windows }} == 'disabled' ]]; then
+          set -euxo pipefail
```

Comment:
This is required for now, some windows pre-scripts have errors, however build works. Need to cleanup windows scripts in vision and audio side